File: 프로그래머스/Level1/이상한 문자 만들기.py
# https://programmers.co.kr/learn/courses/30/lessons/12930

def solution(s):
    answer = ''
    idx = 0
    for i in range(len(s)):
        if s[i] == ' ':
            idx = 0
            answer += ' '
            continue
        if idx % 2 == 0:
            answer += s[i].upper()
        else:
            answer += s[i].lower()
        idx += 1
    return answer

File: 프로그래머스/Level1/이상한 문자 만들기.js
// https://programmers.co.kr/learn/courses/30/lessons/12930

function solution(s) {
	let answer = '';
	let temp = 1;
	s.split('').forEach((a, idx) => {
		if (a === ' ') {
			temp = 1;
			answer += a;
		} else if (temp % 2 === 0) {
			answer += a.toLowerCase();
			temp++;
		} else if (temp % 2 !== 0) {
			answer += a.toUpperCase();
			temp++;
		}
	});
	return answer;
}
